lib/isafw/isafw.py

```diff
@@ -30,8 +30,12 @@
 
 import sys
 import traceback
-import isafw.isaplugins as isaplugins
-
+try:
+    # absolute import
+    import isafw.isaplugins as isaplugins
+except ImportError:
+    # relative import when installing as separate modules
+    import isaplugins
```

Comment:
You also need to remove the "import from __future__" line to make the relative "import isaplugins" work.